mirage/config.js

```diff
@@ -17,7 +17,20 @@ export default function() {
     }
   });
 
-  this.get('/articles');
+  this.get('/articles', (schema, request) => {
+    const allArticles = schema.articles.all(),
+      params = request.queryParams,
+      limit = parseInt(params.limit),
+      page = parseInt(params.offset) / limit,
+      start = page * limit,
+      end = start + limit,
+      newArticles = allArticles.models.slice(start, end),
+      newSchema = {};
+    newSchema.articles = newArticles;
+    newSchema.articlesCount = allArticles.length;
```

Comment:
Running in Mirage this seems to cause a warning:
`warn.js:45 WARNING: Encountered "articlesCount" in payload, but no model was found for model name "articles-count" (resolved model name using (unknown).modelNameFromPayloadKey("articlesCount"))`
It seems that articlesCount isn't being set into `meta` and Ember Data is instead trying to serialize this as a model